cli/main.rs

```diff
@@ -217,35 +133,129 @@ fn types_command() {
   println!("{}", content);
 }
 
-fn fetch_or_info_command(
-  flags: DenoFlags,
-  argv: Vec<String>,
-  print_info: bool,
-) {
-  let (mut worker, state) = create_worker_and_state(flags, argv);
+fn print_cache_info(worker: Worker) {
+  let state = worker.state;
+
+  println!(
+    "{} {:?}",
+    ansi::bold("DENO_DIR location:".to_string()),
+    state.dir.root
+  );
+  println!(
+    "{} {:?}",
+    ansi::bold("remote modules cache:".to_string()),
+    state.dir.deps_cache.location
+  );
+  println!(
+    "{} {:?}",
+    ansi::bold("TypeScript compiler cache:".to_string()),
+    state.dir.deps_cache.location
```

Comment:
Should be `state.dir.gen_cache.location`

Reply:
:+1: Thanks